astroquery/atomic/core.py

```diff
@@ -27,7 +29,7 @@ class AtomicLineListClass(BaseQuery):
 
     def __init__(self):
         super(AtomicLineListClass, self).__init__()
-        self._default_form_values = None
+        self._default_form_values_ = None
```

Comment:
why the trailing underscore?

Reply:
because this is already a hidden property (`_default_line_form_values` is now a property).  It should be private.  Maybe `__default_form_values` is better?

Reply:
yes, I think it's double at the front, but no trailing for private, and simple underscore for protected properties (see in the middle of this section of PEP8: https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles), and no such thing as leading and trailing single :)